README.md

```diff
@@ -1,2 +1,6 @@
-# decentralized-open-science
-Decentralized Open Science
+# Decentralized Open Science
+
+The Decentralized Open Science Subgroup, consisting of Marco Beck, Cornelius Ihle, and Dennis Trautwein, employs decentralized information technology to foster the open science movement.
+Open Science, as, e.g., described in the [twelfe Vienna principles](https://viennaprinciples.org/) describes noble goals.
+However, there are many incentives to abstain from doing Open Science or only to pretend to work openly in practice.
+We believe that decentralization and the shift from predefined fixed extrinsically motivated incentive mechanisms to inherently scientific incentives is a critical factor in creating a self-sustaining open science ecosystem.
```

Comment:
`decentralization` is a loaded term. We should be specific in which aspects should get decentralized.